DTIPrep/DTIPrepRegister.pl

```diff
@@ -918,20 +911,19 @@ sub checkPreprocessFiles {
 =head3 checkPostprocessFiles($dti_file, $DTIrefs, $mri_files)
 
 Function that checks if all postprocessing files (from DTIPrep or
-mincdiffusion) are present in the file system.
+C<mincdiffusion>) are present in the file system.
```

Comment:
one line above, should DTIPrep be in C<>? to be consisrtent with 2 lines later

Reply:
sure. Done